ThunkLibs/HostLibs/CMakeLists.txt

```diff
@@ -15,27 +15,27 @@ endif()
 # - custom command: Main build step that runs the thunk generator on the given interface definition
 # - libxyz-interface: Target for IDE integration (making sure libxyz-interface.cpp shows up as a source file in the project tree)
 # - libxyz-deps: Interface target to read include directories from which are passed to libclang when parsing the interface definition
-function(generate NAME SOURCE_FILE)
+function(generate NAME SOURCE_FILE GUEST_BITNESS)
```

Comment:
We don't need to compile interface definitions per-bitness, do we?

Reply:
I kept it in there just in-case we do need it for something. Not sure if it will matter long-term.